pkg/cmd/grafana-cli/commands/datamigrations/encrypt_datasource_passwords.go

```diff
@@ -62,7 +62,7 @@ func EncryptDatasourcePaswords(c utils.CommandLine, sqlStore *sqlstore.SqlStore)
 }
 
 func migrateColumn(session *sqlstore.DBSession, column string) (int, error) {
-	var rows []map[string]string
+	var rows []map[string][]byte
```

Comment:
This was the first fix, scanning a row containing a row with `secure_json_data=null` to a string throwed error. Tried to create a struct for this, but tricky since it's operating on dynamic password columns so skipped that and using byte arrays instead of string.